clear_voices.py

```diff
@@ -0,0 +1,14 @@
+# To be moved to tools(or some other directory later) directory
```

Comment:
I think this is a simple file for testing, right? This code should be in the class later?